crates/chronicle/src/codegen/model.rs

```diff
@@ -1221,52 +1366,69 @@ pub mod test {
         name: evidence
         attributes:
           CMSId:
+            doc: ~
```

Comment:
Bit tedious but if we're feeding in all these doc strings anyway, maybe make them different (but still perfunctory) to make sure that none get mixed up somehow in the output - generally a good idea with tests to vary input values where that can cheaply increase specificity.

Reply:
These are `None` though, because they're representing a domain model without any doc strings added.

Reply:
Aha, thank you, I didn't realize that `~` signifies `None`.